src/tools/clippy/tests/ui/crashes/ice-6252.stderr

```diff
@@ -6,11 +6,14 @@ LL |     _n: PhantomData,
    |
 help: consider importing one of these items
    |
-LL | use core::marker::PhantomData;
+LL + use core::marker::PhantomData;
```

Comment:
@estebank I mean these `use` suggestions. Maybe we're not using the dummy span here though?

Reply:
These are not using DUMMY_SP, they look at the root of the current `mod`, if an appropriate span isn't found, these are supposed to emit a note (and I believe that happens in a couple of places).

Reply:
Why is this one not fixed given your last commit?

Reply:
It was in clippy: fixed